main.py

```diff
@@ -383,9 +395,7 @@ def export_assets(request):
 
         try:
             output_config = asset_v1.OutputConfig()
-            output_config.gcs_destination.uri_prefix = (
-                f"{OUTPUT_BUCKET}/asset_export_v1/{content_type}"
-            )
+            output_config.gcs_destination.uri_prefix = f"{OUTPUT_BUCKET}/asset_export_v1/{content_type}"  # use timestamped bucket name
```

Comment:
how is this different than before?

Reply:
Formatting is different from updating a different PR, this will go back to how it was before when I finish cleanup.